projects/plugins/jetpack/_inc/client/writing/composing.jsx

```diff
@@ -203,6 +205,15 @@ export class Composing extends React.Component {
 						{ __( 'Discover Jetpack tools in the block editor', 'jetpack' ) }
 					</CompactCard>
 				</>
+			),
+			aiAssistantLink = (
+				<CompactCard className="jp-settings-card__configure-link">
+					<a href={ `${ this.props.siteAdminUrl }admin.php?page=my-jetpack#/jetpack-ai` }>
+						{ __( 'Learn more about all Jetpack AI features', 'jetpack' ) }
+					</a>
+					{ /* TODO: remove this Chip once it's not longer "new" */ }
```

Comment:
Do you have a follow-up issue opened already to clean that up, or an approximate idea of when you think it should be removed so we can create an issue to remind us?

Reply:
Good question, how long "new" is actually "new" to promote it as such. @ederrengifo can/should we establish already how long this Chip should be there so I can update the [issue](https://github.com/Automattic/jetpack/issues/38461) to remove it? (asking you since seems more a marketing/design decision than a functional one)